src/ThreeEditor/util/Beam.ts

```diff
@@ -41,7 +42,9 @@ const _default = {
 		id: 2,
 		a: 12,
 		z: 6
```

Comment:
```suggestion
		a: 1,
		z: 1
```

to be consistent with `id = 2` we should set a = 1 and z = 1 (as we are using proton)